### PR TITLE
Part of #18714 Replace deprecated design system typography consts with enums in:  'srp-input'

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
This PR is a part of the issue #18714 Replace deprecated design system typography consts with enums in: 
**ui\components\app\srp-input\srp-input.js**

Part of #18714 

### Before
Screenshot (93)

### After
Screenshot (95)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by brad-decker — DISMISSED** [review]

File: ui/components/app/srp-input/srp-input.js [hunk 1]
```diff
@@ -129,13 +129,14 @@ export default function SrpInput({ onChange, srpText }) {
...
       <label className="import-srp__srp-label">
-        <Typography
-          align={TEXT_ALIGN.LEFT}
-          variant={TypographyVariant.H4}
-          fontWeight={FONT_WEIGHT.BOLD}
+        <Text
+          align={TextAlign.Left}
+          variant={TextVariant.headingSm}
+          as="h4"
+          fontWeight={FontWeight.Bold}
```
Comment (georgewrmarshall) [inline ui/components/app/srp-input/srp-input.js:136]: nit: shouldn't be needed for `headingSm` variant as it is already bold
```suggestion
```

**Review by georgewrmarshall — DISMISSED** [review]
Left one nit otherwise LGTM!

**Review by georgewrmarshall — COMMENTED** [review]
Need to fix lint issue then I can approve. Nice work!

**Review by georgewrmarshall — APPROVED** [review]
LGTM

**Review by garrettbear — APPROVED** [review]

**Pull request merged**